### ISSUE TITLE
Resolve urls, not just bibcodes?

### ISSUE THREAD

**Issue by Gabriel-p** [ryanvarley/adsbibtex#8]
It'd be great if the code could resolve not just bibcodes, but also urls pointing to a specific ADS article.

For a bonus, it could also resolve arXiv's urls as well as many as possible well-known journals, e.g.: resolve http://stacks.iop.org/0004-637X/731/i=1/a=22?key=crossref.e9087a772f3df52358f8892c9ec90a24

**Comment by ryanvarley** [comment]
yeah, i've been thinking of switching away from the ads api because of rate limits and #2. Arxiv would be another nice to have but im not sure when i could get round to it.

Feel free to add it and send a pull request if you can!

**Comment by Gabriel-p** [comment]
I'll try to write a quick and dirty (mostly dirty :) function to resolve urls as soon as I can.

This (abandoned?) package apparently was able to do it: https://github.com/jonathansick/ads_bibdesk
